app/controllers/agents_controller.rb

```diff
@@ -18,15 +18,17 @@ def show
   end
 
   def ajax_agents
-    filters = { name: params[:name] }
-    filters[:agentType] = params[:agent_type] if params[:agent_type]
-    @agents = LinkedData::Client::Models::Agent.all(filters)
+    @agents = LinkedData::Client::Models::Agent.all
     agents_json = @agents.map do |x|
       {
         id: x.id,
         name: x.name,
         type: x.agentType,
-        identifiers: x.identifiers.map { |i| "#{i.schemaAgency}:#{i.notation}" }.join(', ')
+        identifiers: x.identifiers.map { |i| "#{i.schemaAgency}:#{i.notation}" }.join(', '),
```

Comment:
The more you add filters here, the more the filter results will not be precise. For example I search syphax, I will get maybe the first agent with their homepage containing `syphax` where I was only interested in the agent with the name `syphax`.

My proposition is to search only by name, identifiers and acronym